yarn.lock

```diff
@@ -0,0 +1,1107 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Yarn is currently being considered in https://github.com/sanctuary-js/sanctuary/issues/265